docs/azure/nRFCloudLocationServices.rst

```diff
@@ -0,0 +1,39 @@
+.. _azure-nrf-cloud-location-services:
+
+nRF Cloud Location Services
+###########################
+
+Cell Location Service
+*********************
+
+You can enable the :ref:`cell geolocation lookup <app-cellgeolocation>` for single cells and neighboring cell reports for the application using `nRF Cloud's Cell Location Service API <https://api.nrfcloud.com/v1#tag/Cell-Position>`_ in your deployment.
+
+To use nRF Cloud's Cell Location Service API, set the ``enableNrfCloudCellLocation`` parameter to ``true`` and set the ``nrfCloudTeamId`` parameter to your team ID when deploying the solution.
+
+.. parsed-literal::
+   :class: highlight
+
+    az deployment group create \\
+        --resource-group ${RESOURCE_GROUP:-nrfassettracker} \\
+        --mode Complete \\
+        --name enable-unwiredlabs \\
+        --template-file azuredeploy.json \\
+        --parameters \\
+            appName=${APP_NAME:-nrfassettracker} \\
+            appRegistrationClientId=$APP_REG_CLIENT_ID \\
+            b2cTenant=${B2C_TENANT:-nrfassettrackerusers} \\
+            keyVaultName=${APP_NAME:-nrfassettracker} \\
+            enableNrfCloudCellLocation=true \\
+            nrfCloudTeamId=*your team ID*
+
+This command enables the `geolocateCellFromNrfCloud` function to resolve cells.
```

Comment:
Same question as above. How do we want to display the function name?

Reply:
```suggestion
This command enables the ``geolocateCellFromNrfCloud`` function to resolve cells.
```